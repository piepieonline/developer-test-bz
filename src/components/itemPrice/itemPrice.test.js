import React from 'react';
import ReactDOM from 'react-dom';
import ItemPrice from './itemPrice';

it('renders the expected HTML (no sale price)', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ItemPrice price="10.00" />, div);

    expect(div).toMatchSnapshot();

    ReactDOM.unmountComponentAtNode(div);
});

it('renders the expected HTML (sale price)', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ItemPrice price="10.00" salePrice="5.00" />, div);

    expect(div).toMatchSnapshot();

    ReactDOM.unmountComponentAtNode(div);
});