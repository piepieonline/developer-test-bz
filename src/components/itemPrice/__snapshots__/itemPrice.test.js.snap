// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the expected HTML (no sale price) 1`] = `
<div>
  <div
    class="price-container"
  >
    <span
      class="price"
    >
      $
      10.00
    </span>
  </div>
</div>
`;

exports[`renders the expected HTML (sale price) 1`] = `
<div>
  <div
    class="price-container"
  >
    <span
      class="price"
    >
      $
      5.00
    </span>
    <span
      class="price on-sale"
    >
      $
      10.00
    </span>
  </div>
</div>
`;
